Add cookie banner at bottom of home page

kfranqueiro opened the pull request:
This fails both Focus not Obscured criteria by placing a cookie banner at the bottom left of the home page in a way that should be guaranteed to cover one or more of the links in the footer. The banner is intentionally placed after the footer in the document order to attempt to highlight the issue.

The banner dismisses upon clicking either button, and will not be shown again (unless you clear localStorage).

(Regardless of what the banner says, the code in this repo never stores cookies.)

Note: This PR removes the link around "Unsplash" in the footer to avoid needing to work around it in the fixed version.

fstrr reviewed (approved):
The dialog also fails Reflow as the top of it is obscured when the browser is zoomed up to 400%.

kfranqueiro commented:
> The dialog also fails Reflow as the top of it is obscured when the browser is zoomed up to 400%.

Thanks for pointing this out. Added a failure note, and implemented fix for the fixed version.
